frontend/app/routes/_gcweb-app._index.tsx

```diff
@@ -36,23 +36,23 @@ export default function Index() {
         {t('common:index.page-title')}
       </h1>
       <p>
-        Welcome {userInfo.firstName} {userInfo.lastName}
+      {t('common:index.welcome')} {userInfo.firstName} {userInfo.lastName}
```

Comment:
This page should only use translations from its own namespace, and *maybe* the `gcweb` one (for common error messages).

Every page's translations should be isolated, even if it means repeating some strings in different .json files. The reason is that if we have a `common.json` file that contains a string, and that string is used by multiple pages, when we update that string all of those pages are affected.